lib/grizzly/commands/table.ex

```diff
@@ -443,14 +499,35 @@ defmodule Grizzly.Commands.Table do
     @no_validate [:multi_channel_get_command_encapsulation]
 
     defmacro __after_compile__(_, _) do
-      for {table_name, {command_module, _}} <- @table, table_name not in @no_validate do
+      for {table_name, {command_module, opts}} <- @table, table_name not in @no_validate do
         {:ok, command} = command_module.new([])
 
         if table_name != command.name do
           raise """
           The command named #{inspect(table_name)} does not match the name returned from `#{inspect(command_module)}.new/1`.
           """
         end
+
+        case opts[:handler] do
+          {WaitReport, handler_opts} ->
+            if handler_opts[:complete_report] == nil do
+              raise """
+              The command named #{inspect(table_name)} has a handler of WaitReport with no `complete_report` option.
+              """
+            end
+
+            # ensure handler_opts[:complete_report] is an atom, but not a module name
+            if Atom.to_string(handler_opts[:complete_report]) =~ ~r/[A-Z]/ do
```

Comment:
```suggestion
            if module?(handler_opts[:complete_report]) do
```

ChatGPT came up with the following which I thought was kind of clever. But mostly I wanted to remove the regex:

```
  defp module?(atom) when is_atom(atom) do
    function_exported?(atom, :__info__, 1)
  end
```

Reply:
That's a lot cleverer than what it came up with when I asked. 🤣

Reply:
🤣